packages/node/src/builders/execute/execute.impl.ts

```diff
@@ -54,16 +55,21 @@ export function nodeExecuteBuilderHandler(
       return startBuild(options, context).pipe(
         concatMap((event: NodeBuildEvent) => {
           if (event.success) {
-            return restartProcess(event.outfile, options, context).pipe(
-              mapTo(event)
-            );
+            if (options.restart) {
+              return restartProcess(event.outfile, options, context).pipe(
+                mapTo(event)
+              );
+            } else if (!subProcess) {
```

Comment:
This is checked in `runProcess` so I think we're safer with an `else` here. They'll get an error that it was already running which I think rarely happens in practice.

Reply:
Yes, but there throws an error that the process is already running.

Reply:
Maybe I don't understand the use case. What is the issue with throwing an error?

Reply:
Throwing an error, stops the application.
```
Already running
Error: Already running
error Command failed with exit code 1.
```

Reply:
This block is for `!options.watch` right? There _should not_ be an existing process for any reason.

Reply:
I'm gonna close that conversation since is outdated.